lib/WP_Auth0_Embed_Widget.php

```diff
@@ -31,6 +31,7 @@ public function form( $instance ) {
 		wp_enqueue_script( 'wpa0_admin' );
 		wp_enqueue_style( 'media' );
 		require WPA0_PLUGIN_DIR . 'templates/a0-widget-setup-form.php';
+		return 'form';
```

Comment:
Can you explain why this is needed?

Reply:
function `form()` returns a string - bad thing originating from core

Reply:
Oh wow ... TIL 😄